packages/wonder-blocks-dropdown/components/action-menu.md

```diff
@@ -267,4 +267,41 @@ class ControlledActionMenuExample extends React.Component {
 }
 
 <ControlledActionMenuExample />
-```
\ No newline at end of file
+```
+
+### Example: ActionMenu with custom opener
+
+In case you need to use a custom opener, you can use the `opener` property to achieve this:
+
+```js
+import {ActionMenu, ActionItem, OptionItem, SeparatorItem} from "@khanacademy/wonder-blocks-dropdown";
+import {View} from "@khanacademy/wonder-blocks-core";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import {icons} from "@khanacademy/wonder-blocks-icon";
+import {LabelMedium} from "@khanacademy/wonder-blocks-typography";
+
+<ActionMenu
+    disabled={false}
+    menuText="Betsy Appleseed"
+    testId="teacher-menu"
+    opener={(eventState) => (
+        <LabelMedium
+            onClick={()=>{console.log('custom click!!!!!')}}
+        >This is a label</LabelMedium>
+    )}
```

Comment:
I think it would be good too document the `eventState` or cross-link to other docs if they exist.  Also, it would be good to make use of it here to show different states in this example.